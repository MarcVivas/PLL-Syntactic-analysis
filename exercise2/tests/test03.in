// Test assignment and register usage
a = 3;
b = 4;
c = a + b;
c * 2;
a = c / 2;
a;
b;
c;

a = c;
a;
