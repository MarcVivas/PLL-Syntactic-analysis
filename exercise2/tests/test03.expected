14
3
4
7
7
a = 7
b = 4
c = 7
